backend/core/views.py

```
@@ -1107,11 +1107,21 @@ def risk_assessment_pdf(self, request, pk):
             context = RiskScenario.objects.filter(
                 risk_assessment=risk_assessment
             ).order_by("ref_id")
+            general_settings = GlobalSettings.objects.filter(name="general").first()
```

Ruff (0.8.2)

`GlobalSettings` may be undefined, or defined from star imports

(F405)